go/caddy/log-tailer/handlers.go

```diff
@@ -0,0 +1,245 @@
+package logtailer
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/inverse-inc/packetfence/go/sharedutils"
+)
+
+func pflog(log string) string {
+	return "/usr/local/pf/logs/" + log
+}
+
+func (h LogTailerHandler) optionsSessions(c *gin.Context) {
+	c.JSON(http.StatusOK, gin.H{
+		"meta": gin.H{
+			"filter": gin.H{
+				"type":        "string",
+				"required":    false,
+				"default":     nil,
+				"placeholder": nil,
+			},
+			"filter_is_regexp": gin.H{
+				"type":        "string",
+				"required":    false,
+				"default":     nil,
+				"placeholder": false,
+			},
+			"files": gin.H{
+				"type":        "array",
+				"required":    true,
+				"placeholder": nil,
+				"default":     nil,
+				"item": gin.H{
+					"type":        "string",
+					"required":    true,
+					"placeholder": nil,
+					"default":     nil,
+					"allowed": []gin.H{
```

Comment:
Most of the files are defined here https://github.com/inverse-inc/packetfence/blob/devel/lib/pf/constants/syslog.pm maybe we can create a pfconfig that expose them and use it.